deploy/scripts/shared.sh

```
@@ -38,6 +38,14 @@ render_service() {
                 cp "$container_file" "${QUADLET_FILES_OUTPUT_DIR}/${base_filename}"
             fi
         done
+        
+        # Process .service files for systemd services
+        for service_file in "${source_dir}/flightctl-${service_name}"/*.service; do
+            if [[ -f "$service_file" ]]; then
+                local base_filename=$(basename "$service_file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)